### DISCUSSION TITLE
Hey, add whitelist that doesnt flag people as cheating

### DISCUSSION THREAD

**Discussion by bozogithubbozo** [Scythe-Anticheat/Scythe-Anticheat#88] [Help and Support] [answered]
Hey, me and friend wanted to make smp but both of me uses client that are hacking

Please add whitelist to your anticheat

Have a good day

@MrDiamond64

**Comment by MrDiamond64** [comment, marked as answer]
This already exists, edit config.js file in the addon. Also, please dont mention me.

**Comment by bozogithubbozo** [comment]
Oh sorry
Dear diamond

**Comment by bozogithubbozo** [comment]
also diamond, how do i whitelist 2 peoples? diamond

**Comment by bozogithubbozo** [comment]
wtf

**Comment by bozogithubbozo** [comment]
why is this now discussing

**Reply by MrDiamond64** [reply]
> why is this now discussing

Because its better suited for here.

**Comment by bozogithubbozo** [comment]
fucker

**Comment by bozogithubbozo** [comment]
i hate you retard
